web/src/components/forms/components/schemas-databio-dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import Select, { ValueType } from 'react-select';
+
+interface Template {
+  name: string;
+  project: string;
+  description: string;
+  url: string;
+}
+
+interface Props {
+  onSelectTemplate: (template: Template) => void;
+  selectedSchema: ValueType<{ label: string; value: string }> | null;
+}
+
+const SchemaDropdown: React.FC<Props> = ({ onSelectTemplate, selectedSchema }) => {
+  const [templates, setTemplates] = useState<Template[]>([]);
+
+  useEffect(() => {
```

Comment:
Wondering if instead of `useEffect` we could use the `useSchemas` hook I made? I think its here: https://github.com/pepkit/pephub/blob/dev/nathan/web/src/hooks/queries/useSchemas.ts

The syntax should be something like:

```tsx
import {useSchemas} from '...' // where I put it...
const { data: schemas, isLoading } = useSchemas();
```

Right now we are just hitting the `eido/schemas` endpoint to grab the list of schemas, but we can change that if necessary:

```ts
const fetchSchemas = async () => {
  const { data } = await axios.get<SchemaResults>(`${API_BASE}/eido/schemas`);
  return data;
};
```

Reply:
Ok, yeah I was wondering since the schemas will now live at https://schema.databio.org/list.json the eido/schemas endpoint would need to grab the list from there rather than the internal schemas.yaml file that's in pephub. Not sure how to change to that yet.